apps/mux/frontend/public/index.html

```diff
@@ -5,6 +5,5 @@
   </head>
   <body>
     <div id="root"></div>
-    <script type="text/javascript" src="./index.tsx"></script>
```

Comment:
just out of curiosity here, why is this being removed?

Reply:
Good catch that looks like a mistaken deletion while experimenting (though interestingly the app still runs just fine without that line 🤔 )

Reply:
E.g. I'm comparing to https://github.com/contentful/apps/blob/master/apps/ai-image-generator/frontend/public/index.html which does not have this line but  https://github.com/contentful/apps/blob/master/apps/microsoft-teams/frontend/index.html does

Reply:
After chatting internally we're inclined to keep this deletion since it's not needed (nor very clear what if any purpose it would serve).